src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/DatabaseErrorPageExtensions.cs

```diff
@@ -26,7 +25,7 @@ public static IApplicationBuilder UseDatabaseErrorPage(this IApplicationBuilder
                 throw new ArgumentNullException(nameof(app));
             }
 
-            return app.UseMiddleware<DatabaseErrorPageMiddleware>();
+            return app.UseDatabaseErrorPage(new DatabaseErrorPageOptions());
```

Comment:
It looks like `MigrationsEndPointMiddleware` was never wired up on this code path. The error page would show up, but the endpoint to apply migrations was never added.